tests/decorators/test_decorators.py

```diff
@@ -201,6 +201,50 @@ def dummy():
                                          "connection": "dummy_conn"
                                      }]})
 
+    def test_queue_trigger(self):
+        app = self.func_app
+
+        @app.on_queue_change(arg_name="req", queue_name="dummy_queue",
+                             connection="dummy_conn")
+        def dummy():
+            pass
+
+        func = self._get_func(app)
```

Comment:
Can you please change the name of this helper function - calling it `_get_user_function`?

Reply:
updated as per recommendation. Thanks.